spec/rubocop/cop/rails/where_missing_spec.rb

```diff
@@ -0,0 +1,207 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::WhereMissing, :config do
+  context 'Rails 6.1', :rails61 do
+    it 'registers an offense when using `left_joins(foo).where(foos: {id: nil})`' do
```

Comment:
If we are including the code in the example, perhaps include the column in the argument
```suggestion
    it 'registers an offense when using `left_joins(:foo).where(foos: {id: nil})`' do
```
Same goes for the rest of the examples

Reply:
Oops, you are right. Thank you so much. I fixed it.